venia-integration-tests/src/fixtures/wishlist/multipleWishlist/categoryPageCreateWishlist2.json

```diff
@@ -0,0 +1,11 @@
+{
+    "data": {
+        "createWishlist": {
+            "wishlist": {
+                "id": "32",
```

Comment:
These don't actually get created on the backend so these `id`s don't need to constantly be updated, right?

Reply:
No, it does not needed. As soon as page reloads or test is complete the data is wiped.